### ISSUE TITLE
[v2] Send-MgUserMail throws error even though it succeeds

### ISSUE THREAD

**Issue by jasonjoh** [microsoftgraph/msgraph-sdk-powershell#1834]
The command results in the following error, but the email is sent.

```powershell
Send-MgUserMail_Send: The server responded with an unrecognized response, Status: Accepted
```

A `202 Accepted` is the expected response for the [send mail API](https://learn.microsoft.com/graph/api/user-sendmail?view=graph-rest-1.0&tabs=http).

Version: 2.0.0-preview5

To reproduce:

```powershell
Connect-MgGraph -Scopes "Mail.Send" -UseDeviceAuthentication

$context = Get-MgContext

$sendMailParams = @{
    Message = @{
        Subject = "Testing Microsoft Graph"
        Body = @{
            ContentType = "text"
            Content = "Hello world!"
        }
        ToRecipients = @(
            @{
                EmailAddress = @{
                    Address = $context.Account
                }
            }
        )
    }
}

Send-MgUserMail -UserId $context.Account -BodyParameter $sendMailParams
```

**Comment by peombwa** [comment]
Thanks for bringing this to our attention.

The status code mismatch is due to a bug in the OData to OpenAPI converter - https://github.com/microsoft/OpenAPI.NET.OData/issues/310.

To avoid blocking customers, we will update the local OpenAPI documents to ensure all success status codes are defined as 2XX status code range.

**Comment by PowerBugi** [comment]
Hello,

Thank you @jasonjoh for the publication, but I would like to add an element :

Indeed there is an error message but the mail is sent, however, **the attachment is not sent !**

So for example if is use [this](https://learn.microsoft.com/en-us/graph/api/user-sendmail?view=graph-rest-1.0&tabs=powershell#example-3--create-a-message-with-a-file-attachment-and-send-the-message) :

```powershell
$params = @{
	Message = @{
		Subject = "TEST MAIL !"
		Body = @{
			ContentType = "Text"
			Content = "hello !"
		}
		ToRecipients = @(
			@{
				EmailAddress = @{
                    Address = "fotechnical@arkema.com"
				}
			}
		)
		Attachments = @(
			@{
				"@odata.type" = "#microsoft.graph.fileAttachment"
				Name = "attachment.txt"
				ContentType = "text/plain"
				ContentBytes = "SGVsbG8gV29ybGQh"
			}
		)
	}
}

Send-MgUserMail -UserId "test@domain.com" -BodyParameter $params
```

It will work (send the attachment) in V1, but in V2, the mail is sent without the attachment.

I don't know i this information can help... 😄

**Comment by peombwa** [comment]
@PowerBugi, the attachment type was missing in the schema. This has now been fixed in v2.0.0-preview6:
![image](https://user-images.githubusercontent.com/7061532/225449313-955202db-33aa-4319-9646-61f7cce1baad.png)


Closing as fixed by https://github.com/microsoftgraph/msgraph-sdk-powershell/releases/tag/2.0.0-preview6.
